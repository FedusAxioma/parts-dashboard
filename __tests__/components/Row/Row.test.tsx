import { createRenderer } from 'react-test-renderer/shallow';
import React from 'react';
import Row from '../../../src/components/Row';

describe('Row', () => {
	const shallow = createRenderer();

	test('renders component', () => {
		shallow.render(<Row><span>Test</span></Row>);

		expect(shallow.getRenderOutput()).toMatchSnapshot();
	});
});
