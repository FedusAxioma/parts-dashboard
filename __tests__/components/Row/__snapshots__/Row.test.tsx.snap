// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row renders component 1`] = `
<div>
  <span>
    Test
  </span>
</div>
`;
